core/src/main/scala/tofu/Timeout.scala

```diff
@@ -13,7 +13,7 @@ trait Timeout[F[_]] {
 }
 
 object Timeout {
-  def concurrent[F[_]](implicit F: Concurrent[F], timer: Timer[F]): Timeout[F] =
+  implicit def concurrent[F[_]](implicit F: Concurrent[F], timer: Timer[F]): Timeout[F] =
```

Comment:
Can you please add `NonTofu` requirement as it is here https://github.com/TinkoffCreditSystems/tofu/blob/8d9b7b846d7e5109b5055e64b439ad4b6ac52d90/core/src/main/scala/tofu/Fire.scala#L33

Reply:
Done